samples/bluetooth/channel_map_updates/README.rst

```diff
@@ -113,11 +117,47 @@ After programming the sample to both development kits, complete the following st
    On Peripheral Unit
    * Prints the Channel Map if it has been updated from last iteration
 
+Testing in Simulation:
+**********************
+1. Start two instances of the BabbleSim board (one acting as central, one as peripheral):
+
+   .. code-block:: bash
+
+      cd build/zephyr
```

Comment:
~~I hear from @aescolar that bsim doesn't work on windows, so I'm a bit surprised to see this with windows file path types?~~ misinterpreted slashes